using-timescaledb/backup.md

```diff
@@ -331,6 +328,26 @@ end as the recovery will be complete when no further files can be
 found in the archive. See the PostgreSQL documentation on
 [continuous archiving][pg archiving] for more information.
 
+### Restore points [](restore_points)
+
+TimescaleDB natively support a restore point both for single-node and for
+multi-node environments. The restore point can later be used as a recovery target on
+single-node or on each node of the entire multi-node cluster, ensuring that it can be
+restored to a consistent state.
+
+To create a restore point on a single-node [`pg_create_restore_point`][pg_create_restore_point]
+function can be used.
+
+On a multi-node node [`create_distributed_restore_point`][create_distributed_restore_point] function must be used
+instead and executed on the access-node. In order to achieve synchronization across the multi-node cluster, this function ensures
+that the restore point created on the access node is synchronized with each data node. Care should be taken to ensure
+third-party solutions properly quiesce the environment before backing up, so that the
+restore point used across nodes does not have outstanding transactions running on the data nodes.
```

Comment:
This should be clarified about what is needed here.  I thought the restore point would take a lock.  This makes it sound like something is needed for _correctness_, or are you just worried about lock delay / deadlock?  Can this be clarified?

Reply:
The basic ideas here is that the function that creates distributed restore point can only guarantee consistency while using TimescaleDB remote transactions. If user have some custom logic and running transactions on a data node directly at the same time (basically without access node knowledge), there are no consistency guarantees for such case.